script.js

```diff
@@ -1,6 +1,8 @@
 /* Todos: 
-- Implement vertical placing 
 - Implement blocking space around vertical placed elements
+- Write checkPlaced und checkBlocked at beginning of function
```

Comment:
When I have additional thoughts about my commits, I write them down in a separate place and add them as questions to my PR in comments like this. (so that they are not tracked in VCS)